### PR TITLE
fix(@clayui/css): Management Bar Search button should be properly ali…

### LINKED ISSUE #4059: @clayui/css Management Bar Search Button too big in mobile view
Caused by #4005 

mgmt-bar-mobile

### PR DESCRIPTION
**Pull request by pat270**
…gned in mobile view

fixes #4059

### CODE REVIEW

File: packages/clay-css/src/scss/variables/_forms.scss [hunk 1]
```diff
@@ -697,6 +697,8 @@ $input-group-inset-item-btn-monospaced: map-deep-merge(
 	(
 		font-size: 1rem,
 		height: 2rem,
+		margin-bottom: math-sign($input-border-bottom-width),
+		margin-top: math-sign($input-border-top-width),
```
Comment (matuzalemsteles) [inline packages/clay-css/src/scss/variables/_forms.scss:701]: @pat270 this will only affect the `input-group-inset` with `btn-monospaced` right?

Comment (pat270) [reply packages/clay-css/src/scss/variables/_forms.scss:701]: @matuzalemsteles Yes, you can see at https://github.com/liferay/clay/blob/1c3f5839afd8a9881dfc4906ac204c27de887e78/packages/clay-css/src/scss/components/_input-groups.scss#L529

Comment (matuzalemsteles) [reply packages/clay-css/src/scss/variables/_forms.scss:701]: Thank you @pat270!

**Review by matuzalemsteles — APPROVED** [review]
LGTM

**Comment by matuzalemsteles** [comment]
Hey @pat270 it looks like you already sync those changes to cadmin in another PR, so we can safely merge that, right?

**Comment by pat270** [comment]
@matuzalemsteles we can merge this. I didn't know if this would make it first or the other pr, just fixed it in the sync pr since that's where the variable was added.

**Comment by matuzalemsteles** [comment]
@pat270 np, thanks for the clarification! Feel free to merge

**Pull request merged**